UDx-container/README.md

```diff
@@ -135,13 +129,26 @@ The following table describes the environment variables that you can set to prov
 
 ## Compile UDxs
 
-After you [test your UDx container](#testing-the-container), you can develop UDxs in the current working directory on the host machine and compile them in the UDx container. Use the `vsdk-make` script to execute your Makefile and compile your UDx. In a new environment, `vsdk-make` mounts the same directories as the `make test` command.
+After you [test your UDx container](#test-the-udx-container), you can develop UDxs in the current working directory on the host machine and compile them in the UDx container.
 
-The following command passes a file containing [environment variables](#environment-variables):
+Use the `vsdk-make` script to execute your Makefile and compile your UDx. This script behaves exactly like `make`, but it compiles your files in the development environment mounted in the UDx container:
 
-```shell
-$ VSDK_ENV=env-vars-file vsdk-make
-```
+1. Add `UDx-container` repository to the `PATH` so you can execute the `vsdk-*` scripts from your development directory:
+   ```shell 
+   $ export PATH=/path/to/vertica-containers/UDx-container:$PATH
+   ```
+2. Change into your UDx development directory:
+   ```shell 
+   $ cd /path/to/dev-dir
+   ```
+3. Run `vsdk-make` with the required environment variables:
+   ```shell 
+   $ VERTICA_VERSION=<vertica-version> OSTAG=ubuntu vsdk-make TARGET=deb
+   ```
+   Alternatively, use `VSDK_ENV` to pass a file that contains the environment variables:
```

Comment:
"Additionally" not "Alternatively".  The VERTICA_VERSION and OSTAG variables don't work inside that file.  The env file is for env variables INSIDE the docker environment.